docs/pages/careers/design-engineer-x-grid.md

```diff
@@ -0,0 +1,122 @@
+
+We are aiming to make high-end design accessible to the many, empowering developers that lack access to a designer as well as saving time for designers with a source they can trust.
+[According to our last survey](https://mui.com/blog/2021-developer-survey-results/#what-are-your-most-important-criteria-for-choosing-a-ui-library), the quality of the design execution (UX & UI) is the most important criteria used by developers & designers to pick their next UI library.
+
+### Why this is interesting
+
+The advanced components portfolio is still small, with a million interesting and challenging problems to solve.
+
+Our products empower React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.
+
+But while we are [the leading](https://tsh.io/state-of-frontend/#over-the-past-year-which-of-the-following-design-systems-was-your-favorite-go-to-solution) UI design system in the frontend space, the adoption of MUI is only at 25%. More importantly, the challenges of developers and designers to solve when creating UIs go way beyond the ones of design systems.
+We envision a future where MUI X becomes the default toolkit for web developers to create advanced UIs.
+It's why we started Base UI, Joy UI, and MUI Toolpad. Design is key to achieving this goal.
+
+## The role
+
+### What you'll do on a day-to-day basis
+
+You will extend the [Design Engineer](https://mui-org.notion.site/Design-Engineer-09e4cac2e0bb4df3b173458b4b4b5666) responsibilities. Depending on the day, you'll:
+
+- **Contribute to deep, meaningful refactors and feature releases**.
+  MUI X is a complex codebase. Components we've shipped recently, such as the data grid and date pickers have required months of dedicated, careful work.
+- **Design for engineers and their end users**.
+  As an engineer yourself, you are able to understand problems developers face, and you will help them overcome those problems by delivering good DX, and enabling them to create powerful UX for their end users
+  This might involve careful API design, identifying and fixing top bugs, creating easier-to-understand error messages, and writing documentation and blog posts about features you and your team ship.
+- **Collaborate with the community**.
+  Many small, as well as meaningful fixes and features, have been contributed by the community. Your role is to draw the best out of the community — to inspire those across the world to create and contribute through your reviews of their issues and pull requests.
+- **Experiment and play**. Great, unexpected features and heisenbug fixes have come from a number of sources — relentlessly methodical processes of elimination, free-flowing team collaboration, inspiration by adjacent libraries and projects, and difficult-to-explain individual strokes of brilliance. Whatever your preferred style is for creating new things that others might not have thought of, you'll find a welcome home on the team.
```

Comment:
```suggestion
- **Experiment and play**. Great, unexpected features and heisenbug fixes have come from a number of sources — relentlessly methodical processes of elimination, free-flowing team collaboration, inspiration from adjacent libraries and projects, and difficult-to-explain individual strokes of brilliance. Whatever your preferred style is for creating new things that others might not have thought of, you'll find a welcome home on the team.
```

Quick fix, but wow, this is like, beautifully written! 😅 "heisenbug"; "welcome home on the team", phew... 🤣